src/sentry/static/sentry/app/views/issueList/actions.tsx

```diff
@@ -413,8 +450,12 @@ const IssueListActions = createReactClass({
                 {({projects, initiallyLoaded, fetchError}) => {
                   const selectedProject = projects[0];
                   return this.renderResolveActions({
-                    hasReleases: new Set(selectedProject.features).has('releases'),
-                    latestRelease: selectedProject.latestRelease,
+                    hasReleases: selectedProject.hasOwnProperty('features')
+                      ? new Set((selectedProject as Project).features).has('releases')
```

Comment:
```suggestion
                      ? (selectedProject as Project).features.includes('releases')
```
You could shave a new Set() off this 🤷 ⛳

Reply:
ooh good call 🏌️